jp.js

```diff
@@ -696,7 +696,7 @@ matchmaker.GameFull ゲームが満員です。
 matchmaker.NoServersMatchQuery ゲーム更新中
 matchmaker.InvalidGameId ゲームが見つかりません。
 matchmaker.NoAvailableServers サーバーが満員です
-matchmaker.NoMatchingServers No Games Found<br><small>Try changing Quickplay options</small>
+matchmaker.NoMatchingServers ゲームが見つかりませんでした。<br><small>クイックプレイオプションを変更してもう一度試してください。</small>
```

Comment:
Try to keep as close as possible to the original translation.

Can you please remove  `。` from both?